L1Trigger/L1CaloTrigger/python/L1TowerCalibrationProducer_cfi.py

```diff
@@ -0,0 +1,185 @@
+import FWCore.ParameterSet.Config as cms
+
+L1TowerCalibrationProducer = cms.EDProducer("L1TowerCalibrator",
```

Comment:
fillDescriptions would be preferable

Reply:
(for all producers introduced in this PR)

Reply:
Propose to do it in the clean-up PR.